pom.xml

```diff
@@ -33,7 +33,8 @@
 	</scm>
 
 	<properties>
-		<graphql-dgs-codegen-core.version>5.2.6</graphql-dgs-codegen-core.version>
+		<graphql-dgs-codegen-core.version>5.2.7</graphql-dgs-codegen-core.version>
+		<graphql-dgs-codegen-core.version>5.2.7</graphql-dgs-codegen-core.version>
```

Comment:
Is this is added twice?

Reply:
weird. Dunno how that happened. Removed it